service/src/test/resources/ehr/mapper/non_consultation_bundles/procedure-request.xml

```diff
@@ -0,0 +1,32 @@
+<component typeCode="COMP" contextConductionInd="true">
```

Comment:
contextConductionInd should not be present - an output PlanStatement wil never be nested